src/main/java/meteordevelopment/meteorclient/utils/misc/MeteorStarscript.java

```diff
@@ -124,6 +124,12 @@ public static void init() {
             .set("health", () -> Value.number(mc.player != null ? mc.player.getHealth() : 0))
             .set("hunger", () -> Value.number(mc.player != null ? mc.player.getHungerManager().getFoodLevel() : 0))
             .set("speed", () -> Value.number(Utils.getPlayerSpeed()))
```

Comment:
you can make this return Utils.getPlayerVelocity().getHorizontalLength() and remove the getPlayerSpeed() method

Reply:
Added